### ISSUE TITLE
Graphics quality

### ISSUE THREAD

**Issue by madiashley1** [BoiseState/Band3-Java#11]
References #7 

When I download the program from the website, the graphics quality is great.
However, when I run it from NetBeans or from my project's jar file, the graphics quality is not very good. It's a little pixelated. 
I want to find out what the problem is there.

**Comment by madiashley1** [comment]
Resolved: The issue wasn't due to the program itself, but from a difference in screen resolution. Running each version looks slightly pixelated on my laptop screen, but they are each clear on my external monitor.

**Comment by billknowlton** [comment]
Interesting. So did you have to make any changes?

**Comment by madiashley1** [comment]
I didn't make any changes here. I thought there was a graphics issue, but it looks like it was just my screen resolution.
I am working on updating the version of Swing so that I can fix the other formatting issues on branch 8-update-swing, issue #8

**Comment by billknowlton** [comment]
Ok. So will the updated version of Swing improve the GUI?

**Comment by madiashley1** [comment]
Yes! Updating it will allow me to fix the formatting issues, like the ones that we talked about. I can also do more changes/updates after that if needed.
Because of the way it was developed on NetBeans, it requires using the Swing Application Framework and the WYSIWYG editor that goes with that in NetBeans 7. Swing Application Framework is deprecated and not supported in any newer IDEs so I can't use its editor.
I also can't change the code for the formatting manually. So I found a software (JFormDesigner) that helped me convert the .form files into a different format that can be used with either NetBeans or IntelliJ, and that also converts it to javax Swing.
(It was using javax Swing before, but the Swing Application Framework is a library that has some additional functionality)
But it is kind of a big change to go from Swing Application Framework to just using javax Swing, so I'm going through the app and trying to fix the logic on it right now so it will work. This is the only solution that I've found so far for updating the GUI.
